src/components/Dailog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="main_container">
+    <div class="container" v-for="issue in issues" :key="issue.id">
+      <v-alert v-show="showAlert(issue)" class="dailog" type="error"
+        >{{ issue.bName }} to be returned on {{ issue.dueDate }}
+      </v-alert>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  name: "alertDailog",
+  data() {
+    return {
+      user: {},
+      issues: [],
+    };
+  },
+  methods: {
+    async loadData() {
+      let user = localStorage.getItem("user-info");
```

Comment:
Since this is a static data(logged In user will never change), avoid using this line in every component file. Can you put this data in store and then dispatch the action to get logged in use in app.vue. and then just use computed prop to get the loggedInUser(return this.$store.state.loggedInUser).

Reply:
If you need oneTime data example, if you want to get the due date of a book and assume that the data is fetched from this api(v1/boos/:bookId/dueDate), then this doesn't need to be saved in store. And you can directly call the api in that case.